Alignment/OfflineValidation/test/BuildFile.xml

```diff
@@ -1,33 +1,34 @@
 <environment>
-  <bin file="testAlignmentOfflineValidation.cpp" name="testAlignmentOfflineValidation">
-    <flags TEST_RUNNER_ARGS=" /bin/bash	Alignment/OfflineValidation/test test_all-in-one.sh"/>
+  <bin file="testAlignmentOfflineValidation.cpp" name="validateAlignments">
+    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test validateAlignments.sh"/>
     <use name="FWCore/Utilities"/>
   </bin>
-  <bin file="testAlignmentOfflineValidation.cpp" name="testPVValidation">
-    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test testPVValidation.sh"/>
+  <bin file="testAlignmentOfflineValidation.cpp" name="DMRall">
+    <flags PRE_TEST="validateAlignments"/>
+    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test test_unitDMR.sh"/>
     <use name="FWCore/Utilities"/>
   </bin>
-  <bin file="testAlignmentOfflineValidation.cpp" name="testMiscellanea">
-    <flags TEST_RUNNER_ARGS=" /bin/bash Alignment/OfflineValidation/test testMiscellanea.sh"/>
```

Comment:
please keep this test.

Reply:
oh, this came recently from you? I thought it is some kind of old relict

Reply:
Actually there was just one test not already kept